components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/Launcher.java

```diff
@@ -153,19 +153,23 @@ public Launcher(final @Nullable String basePath,
     this.server = new JettyServer(cl, props, args);
   }
 
+  public JettyServer getServer() {
+    return server;
+  }
+
   public void start() throws Exception {
-    start(true);
+    start(true, null);
   }
 
-  public void startAsync() throws Exception {
-    start(false);
+  public void startAsync(@Nullable final Runnable callback) throws Exception {
```

Comment:
Might be worth some javadoc and/or a more descriptive parameter name to explain when this callback gets called.